cmd/capibmadm/utils/utils.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 // GetAccountID returns IBM cloud account ID of API key used.
-func GetAccountID(ctx context.Context, auth *core.IamAuthenticator) (string, error) {
+func GetAccountID(ctx context.Context, _ *core.IamAuthenticator) (string, error) {
```

Comment:
rather I prefer removing the argument itself.

Reply:
Removed.